near/nearcore
# feat: lower data receipt cost

Main goal of this PR is to reduce data receipt cost, due to new estimation.

**Cost is reduced by 4.7 Tgas and 4.2 Ggas per KB.**
Note: the latter one is not going to have big impact, because maximal length of data receipt in mainnet is ~0.5 KB:
```
nearcore % psql postgres://public_readonly:nearprotocol@104.199.89.51/mainnet_explorer
mainnet_explorer=> select max(length(data)) as len from data_receipts;
 len
-----
 567
```
 
Secondary goal is to make protocol configs of all chains the same by default. Doesn't seem significant to make separate PR for this.

Test plan
---------
- Check that cost is lowered
- Check that genesis config is overridden for new protocol version
